prow/overlays/cnv-prod/config.yaml

```diff
@@ -305,10 +305,21 @@ periodics:
         - image: quay.io/thoth-station/integration-tests:v0.5.0
           imagePullPolicy: Always
           command:
-            - ./e2e-test.sh
+            - python3
+            - app.py
           env:
-            - name: E2E_TEST_API
+            - name: THOTH_DEPLOYMENT_NAME
+              value: cnv-prod
+            - name: MAIL_REPORT
+              value: "0"
+            - name: THOTH_USER_API_HOST
               value: api.moc.thoth-station.ninja
+            - name: THOTH_MANAGEMENT_API_HOST
+              value: management.moc.thoth-station.ninja
+            - name: THOTH_AMUN_API_HOST
+              value: amun.moc.thoth-station.ninja
+            - name: THOTH_MANAGEMENT_API_SECRET
+              value: ""
```

Comment:
Do we want to have this setup?

Reply:
oh yes we do have set these up. 
same as the stage as of now :see_no_evil:

Reply:
we can have these value set by configmap and secrets. As this component is not in thoth namespaces, so we can have these setup in prow namespace too explicitly.

Reply:
Do we want to keep running these tests in Prow? I wonder if we could rather deploy integration-tests in a namespace running internally and point it to the prod deployment. It is a manner of a new overlay in integration-tests with adjusted setup. We will not need to maintain integration-tests in prow and we reuse the same logic we already have for test and stage environments (without a need to provide secrets here, have periodic e-mail reports). Does Prow give us any advantage here?

Reply:
I m in favor of running a cronjob, which sends out emails. It has an advantage added to it.
prow basically is just executing the cronjob, so I don't see any added advantage.

Reply:
prow advantage I see is, that it keeps all the logs around, and that we declare all the things wrt prod in one place.